src/mapgen/mapgen.cpp

```diff
@@ -1124,13 +1124,15 @@ bool MapgenParams::saoPosOverLimit(const v3f &p)
 {
 	if (!m_mapgen_edges_calculated)
 		calcMapgenEdges();
-
+/* TODO revert, only for testing PR
```

Comment:
well well

Reply:
?
Ths PR will be helpful if #6998 is merged, however also it fixes the code in 'item entity' that was intended to stop entities entering ignore, but failed to do so. So the PR is useful without #6998 However, testing this PR is difficult if this SAO limit code is active.

Reply:
what I'm saying is: this change should not be in this PR (unless it's still WIP?)

Reply:
This change is to be reverted before merge, and i will, it's there to enable others to test.
The PR is not WIP.